Dockerfile

```diff
@@ -38,13 +38,13 @@ RUN curl --retry 3 --silent --location --remote-name https://get.helm.sh/helm-${
     mkdir -p /tmp/helm && tar -C /tmp/helm -xf helm-${HELM_VERSION}-linux-amd64.tar.gz && \
     install -m 0755 /tmp/helm/linux-amd64/helm /usr/local/bin/helm
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+# Use alpine as minimal base image to package the manager binary
+FROM alpine
```

Comment:
Had to replace this to alpine as git installation is required to build dependencies for kustomize as copying just git binary from builder container is not working.